Framework/Algorithms/src/InterpolatingRebin.cpp

```diff
@@ -54,8 +54,26 @@ void InterpolatingRebin::init() {
                   "logarithmic binning. ");
 }
 
+/**
+ * Overwrites the parent (Rebin) validateInputs
+ * Avoids all bother about binning modes and powers
+ */
+/// Validate that the input properties are sane.
+std::map<std::string, std::string> InterpolatingRebin::validateInputs() {
+  std::map<std::string, std::string> helpMessages;
+
+  // as part of validation, force the binwidth to be compatible with set bin mode
+  MatrixWorkspace_sptr inputWS = getProperty("InputWorkspace");
+  std::vector<double> rbParams = getProperty("Params");
+  try {
+    std::vector<double> validParams = Rebin::rebinParamsFromInput(rbParams, *inputWS, g_log);
```

Comment:
To follow from what Rebin does in it's `validateInputs()`, I think think you need to use the `validParams` to set the `Param` property like `Rebin` does. i.e.
```c
      setProperty("Params", validParams);
```